CHANGELOG.md

```diff
@@ -340,6 +340,7 @@ promiseUtils.create deprecated since version 4.24. Use Promise instead.
 - watchUtils.whenUndefined deprecated since 4.24. Use reactiveUtils.when() instead.
 - watchUtils.whenUndefinedOnce deprecated since 4.24. Use reactiveUtils.whenOnce() instead.
 - watchUtils deprecated since version 4.24. Use reactiveUtils instead.
+- SunLighting.ambientOcclusionEnabled, SunLighting.waterReflectionEnabled, VirtualLighting.ambientOcclusionEnabled, VirtualLighting.waterReflectionEnabled and SceneView.atmosphere are deprecated since 4.27. Adaptive rendering now automatically choses the best visual quality possible and these properties no longer have an effect.
```

Comment:
If they no longer have an effect, it looks like we have removed these props - not just deprecated, no?
Ideally, deprecated props would still work as expected until they are removed in a later release.

Reply:
At least SunLighting.ts still has them? 

```
 /**
   * Indicates whether {@link module:esri/symbols/WaterSymbol3DLayer} display reflections of surrounding terrain,
   * buildings and other 3D objects.
   *
   * @type {boolean}
   * @name waterReflectionEnabled
   * @instance
   * @default false
   * @deprecated since version 4.27. Reflections are automatically shown and this property has no effect.
   */
  @property({ type: Boolean })
  get waterReflectionEnabled(): boolean {
    deprecatedProperty(logger, "waterReflectionEnabled", {
      replacement: "reflections are automatically shown and this property has no effect",
      version: "4.27"
    });
    return this._get("waterReflectionEnabled") ?? false;
  }
  set waterReflectionEnabled(value: boolean) {
    deprecatedProperty(logger, "waterReflectionEnabled", {
      replacement: "reflections are automatically shown and this property has no effect",
      version: "4.27"
    });
    this._set("waterReflectionEnabled", value);
  }
```

Reply:
In this specific case, the default behavior for waterReflectionEnabled has changed from false to true without anyway to change it back. Also, for users who explicitly set it to false, the behavior will now automatically change to true. So, even if the property is still there it is non-functional - not just deprecated. Could we add a note in breaking changes section of the release notes?
https://next.sites.afd.arcgis.com/javascript/latest/release-notes/#breaking-changes